clusterloader2/testing/load/modules/dns-k8s-hostnames.yaml

```diff
@@ -0,0 +1,27 @@
+{{$ENABLE_DNSTESTS := DefaultParam .CL2_ENABLE_DNSTESTS false}}
```

Comment:
Can we have separate flag different from CL2_ENABLE_DNSTESTS? If we use the same flag we are risking that currently passing tests break.

Reply:
Added a new flag `USE_ADVANCED_DNSTEST` to guard the new functionality in `deployment.yaml`, `measurement.yaml` and `dns-k8s-hostnames.yaml`.

We should remove this flag after it's confirmed that it works as intended on a subset of tests, so that this runs for all tests that have `ENABLE_DNSTESTS`.